src/sparsevector.jl

```diff
@@ -47,3 +47,40 @@ function findnz(vec::SparseVector{T}) where T
     V = to_jl_type(T)[vec[idx] for idx in I]
     return (I, V)
 end
+
+# implementation of SparseVector{Bool} as Set{Integer} with Integer length
+struct SparseVectorBool <: SparseVector{Bool}
+    l::Int64
+    s::Set{to_cxx_type(Int64)}
+end
+
+spzeros(::Type{Bool}, n::Base.Integer) = SparseVectorBool(n, Polymake.Set{to_cxx_type(Int64)}())
+
+Base.size(v::SparseVector{Bool}) = (v.l,)
+Base.eltype(::SparseVector{Bool}) = Bool
+
+Base.show(io::IOContext, ::MIME{Symbol("text/plain")}, v::Polymake.SparseVectorBool) = print(io, length(v), "-element SparseVectorBool. `true` for indices:\n", v.s...)
+
+Base.@propagate_inbounds function Base.getindex(V::SparseVector{Bool}, n::Base.Integer)
+    (V.l >= n && n >= 1) || throw(BoundsError(V, n))
+    return in(V, n)
+end
+
+Base.@propagate_inbounds function Base.setindex!(V::SparseVector{Bool}, val, n::Base.Integer)
+    (V.l >= n && n >= 1) || throw(BoundsError(V, n))
+    if val
+        push!(V.s, n)
+    else
+        delete!(V.s, n)
+    end
+    return val
+end
+
+function Base.show(io::IO, ::MIME"text/plain", V::SparseVectorBool)
+    l = V.l
+    print(io, "Incidence vector with $l entries. `true` for:\n")
```

Comment:
```suggestion
    print(io, "Incidence vector with length $l. `true` for:\n")
```
This might also need to be adjusted so that it matches the output for the IncidenceMatrix.

Reply:
i just pushed a small suggestion for this